tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -912,7 +912,7 @@ fdb/test_fdb_mac_expire.py:
   skip:
     reason: "Not supported on this DUT topology."
     conditions:
-      - "topo_type not in ['t0', 'm0', 'mx']"
+      - "topo_type not in ['t0', 'm0', 'mx', 'm1', 'm2', 'm3']"
```

Comment:
Need to revise what is the purpose of this feature

Reply:
This is a FDB testcase. FDB is used for L2 packet forwarding (in VLAN). So we don't need this testcase on M1/M2/M3.